test/unit/query/builder.js

```diff
@@ -9269,7 +9269,7 @@ describe('QueryBuilder', () => {
     }).to.throw('Hint comment must be a string');
   });
 
-  it('Any undefined binding in a SELECT query should throw an error', () => {
+  it.only('Any undefined binding in a SELECT query should throw an error', () => {
```

Comment:
This looks like dev change made it to branch, you probably want to clear up the `only`

Reply:
Indeed